pages/contact.html

```diff
@@ -4,20 +4,23 @@
 <head>
     <title>Mount Sion CBS Secondary School</title>
 
+    <link rel="icon" href="../assets/favicon.ico" type="image/x-icon" />
     <link rel="stylesheet" type="text/css" href="../semantic/out/semantic.min.css">
     <link rel="stylesheet" type="text/css" href="../assets/css/style.css">
-    <script src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8="
-        crossorigin="anonymous"></script>
+    <script
+      src="https://code.jquery.com/jquery-3.1.1.min.js"
+      integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8="
+      crossorigin="anonymous"></script>
     <script src="../semantic/out/semantic.min.js"></script>
-    <link rel="icon" href="../assets/favicon.ico" type="image/x-icon" />
     <script src="//code.jquery.com/jquery-2.2.1.js"></script>
 
 </head>
 
 <body>
 
     <div class="ui divbg container">
-        <img class="ui centered image" src="../assets/images/logo.jpg" height="128px">
+    <div class="ui container">    
```

Comment:
This can also be removed, and don't forget to remove the associated closing div tag.